scripts/upload_model.py

```diff
@@ -47,6 +46,11 @@ def get_config():
         default=constants.SUBNET_UID,
         help="The subnet UID.",
     )
+    parser.add_argument(
+        "--pre_7b_block",
```

Comment:
Should we default this to false, similar to the rest of the code? Or remove this flag and just assume we're post 7b?